frontend/sideMenu/factories/homeItems.factory.js

```diff
@@ -62,7 +62,12 @@
                     showIf: () => user.isAdminOfCurrentInst(),
                     sectionTitle: 'INSTITUIÇÃO',
                     topDivider: true,
-                    onClick: () => $state.go(STATES.MANAGE_INST_EDIT, {institutionKey: getInstitutionKey()}),
+                    onClick: () => {
+                        const state = Utils.selectFieldBasedOnScreenSize(
+                            STATES.MANAGE_INST_EDIT, STATES.MANAGE_INST_MENU, SCREEN_SIZES.SMARTPHONE
```

Comment:
I should detect the mobile state by its name, it doesn't happen. MANAGE_INST_EDIT and MANAGE_INST_MENU sounds like desktop states..